demo/Dockerfile_release

```diff
@@ -22,13 +22,15 @@
 # THE SOFTWARE.
 ##
 
-FROM jenkinsci/workflow-demo:1.8
+FROM jenkinsci/workflow-demo:1.13
+
+USER root
```

Comment:
At some point the `jenkinsci/jenkins` grandparent image started running the command as the `jenkins` user (UID 1000), which is a nice idea but is a major hassle in practice. All sorts of things in this image need to be run as (container) root.